# wgpu-core-deps-windows-linux-android 30.0.1: Kandelo needs wgpu-hal's GLES backend.
--- a/Cargo.toml	2026-10-08 15:41:34
+++ b/Cargo.toml	2026-10-08 15:41:34
@@ -39,7 +39,7 @@
 name = "wgpu_core_deps_windows_linux_android"
 path = "src/lib.rs"
 
-[target.'cfg(any(windows, target_os = "linux", target_os = "android", target_os = "freebsd", target_os = "netbsd"))'.dependencies.wgpu-hal]
+[target.'cfg(any(windows, target_os = "linux", target_os = "android", target_os = "freebsd", target_os = "netbsd", target_os = "kandelo"))'.dependencies.wgpu-hal]
 version = "30.0.0"
 
 [lints.clippy]
